test/index.test.ts

```diff
@@ -0,0 +1,10 @@
+// You can import your modules
+// import index from '../src/index'
```

Comment:
@bkeepers Do you think the suggested [commented out] change of importing from `"src"` (vs. `"lib"`) could lead to an escalation of the issues we've been seeing with TypeScript tests and partially mocked `GitHubAPI` objects?

Reply:
Yeah, once tests are all in TypeScript and testing code that is typed, you'll need to figure out how to use mock objects that are typed.

Reply:
Ping @bkeepers.

Reply:
Ping @bkeepers